luma/core/device.py

```diff
@@ -168,6 +173,90 @@ def display(self, image):
         :param image: Image to display.
         :type image: PIL.Image.Image
         """
-        assert(image.size == self.size)
+        assert image.size == self.size
 
         self.image = self.preprocess(image).copy()
+
+
+class fb(device):
```

Comment:
im not sure if the class name should/can be 2 chars long.

Reply:
Just following how the framebuffer is named in Linux (e.g. `/dev/fb0`)... but happy to change this to something like `luma.core.device.linux_frambuffer`. 

_[I didnt want to call this just `luma.core.device.framebuffer` as it would clash with the namespace of `luma.core.framebuffer`]_

Reply:
`FrameBuffer`.